PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -345,7 +345,7 @@ private extension PanModalPresentationController {
          in the presentation animator instead of here
          */
         containerView.addSubview(presentedView)
-        containerView.addGestureRecognizer(panGestureRecognizer)
+        presentedView.addGestureRecognizer(panGestureRecognizer)
```

Comment:
Required but breaking change here.
IMO this makes more sense, because the background area shouldn't be pull-to-dismissable.